shiny/session/_session.py

```diff
@@ -406,9 +411,18 @@ async def _dispatch(self, message: ClientMessageOther) -> None:
 
         try:
             # TODO: handle `blobs`
-            value = await func(*message["args"])
+
+            # Isolate so that the function can be run without an explicit session.
+            # Let the handler function manage its own session context. (due to modules)
```

Comment:
Change comment to explain that the function should be run on demand, and without throwing an error due to reading a reactive value in a non-reactive function.